portfolio/src/main/webapp/submit.js

```diff
@@ -0,0 +1,6 @@
+//fetch function for submission servlet
+async function submissionFetch() {
+  const completionMessage = await fetch("/submitform");
```

Comment:
I believe `fetch()` by default does a `GET` request. It looks like you don't have a `doGet` to handle it. Also, should this be `/submitForm`?